docs/reference/emails.md

```diff
@@ -346,6 +346,197 @@ We could have technically localized strings this way as well rather than wrap te
 
 The bounce and complaint handling code is in `lib/email/bounces.js` but long-term we want to migrate to the email service’s implementation instead, which was written some time ago but has not been deployed yet. The motivation for moving is partly semantic, because bounce records don’t really belong in the auth db, but also security, because the email service should not need access to the auth db.
 
+### Bounce types
+
+<table>
```

Comment:
Any reason not to use [markdown tables](https://www.markdownguide.org/extended-syntax/#tables)?

Reply:
@vbudhram, sorry! It's now been updated to use markdown tables.